streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -74,16 +74,16 @@
      * Create a new {@code KTable} that consists of all records of this {@code KTable} which satisfy the given
      * predicate.
      * All records that do not satisfy the predicate are dropped.
-     * For each {@code KTable} update the filter is evaluated on the update record to produce an update record for the
+     * For each {@code KTable} update the filter is evaluated on the updated record to produce an updated record for the
      * result {@code KTable}.
      * This is a stateless record-by-record operation.
      * <p>
-     * Note that {@code filter} for a <i>changelog stream</i> works different to {@link KStream#filter(Predicate)
+     * Note that {@code filter} for a <i>changelog stream</i> works differently than {@link KStream#filter(Predicate)
```

Comment:
@joel-hamill  I am not a native speaker. Is "to" or "than" correct?

Reply:
than is preferred